### PR TITLE
Sprite and Outfit Update

### PR DESCRIPTION
**Pull request by CerberusHellHound**
List of update: 
- Fixed holster sprite overlay for MK9
- Added new guns : Classic M1911, USP45, Kriss Vector SMG, Galil (Galil is functionally identical to the SR17)
- Added better shotgun ammo boxes and updated the riot and tactical shotgun lockers
- Replaced Zone Commanders sidearms with an USP45 (The Guard Commander still has his Mateba, and a pistol option in his locker)
- UIU and GOC Rep no longer have body armour on themselves. They are diplomatic envoys, not here to fight the Foundation.
- UIU and GOC Rep got a M1911 and an USP, respectively
- Ethics/O5 Rep got his gyrojet pistol replaced with a M1911 pistol. Why was he armed with an experimental sidearm?

### CODE REVIEW

**Comment by CerberusHellHound** [comment]
Oh also, from previous Commits that weren't pushed earlier 
- Improved number of slots for Medical, Engineering and Research (especialyl junior roles)

File: maps/site53/items/automatic.dm [hunk 1]
```diff
@@ -268,3 +268,72 @@
 	attack_verb = list("stabbed", "chopped", "cut")
 	applies_material_colour = 1
 //	drawsound = 'sound/items/unholster_knife.ogg'
+
+
+
+
+
+
```
Comment (ivanmixo) [inline maps/site53/items/automatic.dm:276]:
```suggestion
```
Normal spacing I beg

File: maps/site53/items/automatic.dm [hunk 2]
```diff
@@ -268,3 +268,72 @@
...
+/obj/item/gun/projectile/automatic/scp/galil/update_icon()
+	..()
+	if(ammo_magazine)
+		icon_state = "galil"
+	else
+		icon_state = "galil-empty"
+	return
+
+
```
Comment (ivanmixo) [inline maps/site53/items/automatic.dm:306]:
```suggestion
```

File: maps/site53/items/guns.dm [hunk 3]
```diff
@@ -82,3 +82,46 @@
 		icon_state = "gyropistolloaded"
 	else
 		icon_state = "gyropistol"
+
+
+
+
+
```
Comment (ivanmixo) [inline maps/site53/items/guns.dm:89]:
```suggestion
```

File: maps/site53/items/guns.dm [hunk 4]
```diff
@@ -82,3 +82,46 @@
...
+	caliber = ".45"
+	silenced = 0
+	fire_delay = 3
+	origin_tech = list(TECH_COMBAT = 2, TECH_MATERIAL = 2, TECH_ILLEGAL = 2)
+	load_method = MAGAZINE
+	magazine_type = /obj/item/ammo_magazine/scp/m1911
+	allowed_magazines = list(/obj/item/ammo_magazine/scp/m1911)
+
+
+
```
Comment (ivanmixo) [inline maps/site53/items/guns.dm:105]:
```suggestion
```

File: maps/site53/items/guns.dm [hunk 5]
```diff
@@ -82,3 +82,46 @@
...
+	load_method = MAGAZINE
+	magazine_type = /obj/item/ammo_magazine/scp/usp45
+	allowed_magazines = list(/obj/item/ammo_magazine/scp/usp45)
+
+
+
+
+
+
+
```
Comment (ivanmixo) [inline maps/site53/items/guns.dm:126]: wew my lad

File: maps/site53/items/projectiles.dm [hunk 6]
```diff
@@ -163,3 +163,45 @@
...
+	origin_tech = list(TECH_COMBAT = 2)
+	mag_type = MAGAZINE
+	caliber = ".45"
+	matter = list(DEFAULT_WALL_MATERIAL = 1500)
+	ammo_type = /obj/item/ammo_casing/c45
+	max_ammo = 30
+	multiple_sprites = 1
+
+
+
```
Comment (ivanmixo) [inline maps/site53/items/projectiles.dm:180]:
```suggestion
```

File: maps/site53/items/projectiles.dm [hunk 7]
```diff
@@ -163,3 +163,45 @@
...
+	icon_state = "magnum"
+	mag_type = MAGAZINE
+	ammo_type = /obj/item/ammo_casing/c45
+	matter = list(DEFAULT_WALL_MATERIAL = 1200)
+	caliber = ".45"
+	max_ammo = 12
+	multiple_sprites = 1
+
+
+
```
Comment (ivanmixo) [inline maps/site53/items/projectiles.dm:193]:
```suggestion
```

File: maps/site53/items/projectiles.dm [hunk 8]
```diff
@@ -163,3 +163,45 @@
...
+	icon_state = "45"
+	mag_type = MAGAZINE
+	ammo_type = /obj/item/ammo_casing/c45
+	matter = list(DEFAULT_WALL_MATERIAL = 1200)
+	caliber = ".45"
+	max_ammo = 7
+	multiple_sprites = 1
+
+
+
```
Comment (ivanmixo) [inline maps/site53/items/projectiles.dm:206]:
```suggestion
```

**Comment by ivanmixo** [comment]
Yeah, this is why you don't make PRs from your main branch, kids

**Comment by CerberusHellHound** [comment]
Because Security Loadout access were readded, I also fiddled around and restricted some gears from Security (no Hawaiawan outfit guard, we're not Hippie station, IMMERSSHUN 🤓).
Also disabled some non lore friendly items such as gears directly mentioning NT, Solgov, ETC. Most of it is largely still available however

**Comment by CerberusHellHound** [comment]
All guards spawn with gears directly in the belt now. Functional.

**Comment by CerberusHellHound** [comment]
Done.

**Comment by CerberusHellHound** [comment]
closing (outdated map data)

**Pull request closed without merging**